solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1163,16 +1211,16 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
     }
 
     /**
-      * @dev returns the adjusted amount of reserve tokens based on the exposure and rate changes
+      * @dev returns the entitled amount of reserve tokens before compensation
```

Comment:
if this returns the initial reserve amount + the fees, that's probably how we should define/describe it

Reply:
Fixed.